packages/block-library/src/comments-pagination/edit.js

```diff
@@ -37,17 +39,18 @@ export default function QueryPaginationEdit( {
 		const innerBlocks = getBlocks( clientId );
 		/**
 		 * Show the `paginationArrow` control only if a
-		 * `QueryPaginationNext/Previous` block exists.
+		 * `CommentsPaginationNext/Previous` block exists.
```

Comment:
```suggestion
		 * Comments Pagination Next/Previous block exists.
```

Reply:
Let's merge first the Comments Pagination Previous. And after that we can update with:
`Comments Pagination Next and/or Previous`

Reply:
Whatever works best. From my perspective, we can land this PR first because the other one still needs to be reviewed 😄